src/Raven.Server/Config/Categories/StorageConfiguration.cs

```diff
@@ -64,6 +64,12 @@ public class StorageConfiguration : ConfigurationCategory
         [ConfigurationEntry("Storage.PrefetchResetThresholdInGb", ConfigurationEntryScope.ServerWideOrPerDatabase)]
         public Size PrefetchResetThreshold { get; set; }
 
+        [Description("Size of journal files in gigabytes for each storage, before forcing a sync and removing the journal files.")]
```

Comment:
Fix the comment size
In general, the entry name should indicate the unit, too. 
`Storage.JournalSizeThresholdInMB`